web/src/components/organisms/Asset/convertAsset.ts

```diff
@@ -0,0 +1,17 @@
+import { Asset, AssetFile, PreviewType } from "@reearth-cms/components/molecules/Asset/asset.type";
+import { Asset as GQLAsset } from "@reearth-cms/gql/graphql-client-api";
+import { uuidToURL } from "@reearth-cms/utils/uuidToURL";
+
+export const convertAsset = (GQLAsset: GQLAsset | undefined): Asset => {
+  return {
+    id: GQLAsset?.id ?? "",
+    fileName: GQLAsset?.fileName ?? "",
+    createdAt: GQLAsset?.createdAt.toString() ?? "",
+    createdBy: GQLAsset?.createdBy?.name ?? "",
+    file: GQLAsset?.file as AssetFile,
+    previewType: GQLAsset?.previewType as PreviewType,
+    projectId: GQLAsset?.projectId ?? "",
+    size: GQLAsset?.size ?? 0,
+    url: GQLAsset?.uuid && GQLAsset?.fileName ? uuidToURL(GQLAsset?.uuid, GQLAsset?.fileName) : "",
+  };
+};
```

Comment:
Isn't it better to return if no GQLAsset and then we can clean it up much more.
Below is just an example and probably has some issues. What do you think?
```
export const convertAsset = (GQLAsset: GQLAsset | undefined): Asset => {
if (!GQLAsset) return;
  return {
    id: GQLAsset.id ,
    fileName: GQLAsset.fileName,
    createdAt: GQLAsset.createdAt.toString(),
    createdBy: GQLAsset.createdBy?.name ,
    file: GQLAsset.file as AssetFile,
    previewType: GQLAsset.previewType as PreviewType,
    projectId: GQLAsset.projectId,
    size: GQLAsset.size,
    url: GQLAsset.uuid && GQLAsset.fileName ? uuidToURL(GQLAsset?.uuid, GQLAsset?.fileName) : "",
  };
};
```

Reply:
Then the return type should be `Asset | undefined` instead of `Asset` and that type change should be done in multiple places. does it worth it?

Reply:
Depends on how it's being used. 
But I'm guessing you are converting queried GQLAssets to Assets that will be displayed in some way (or at least affects the UI in some way). If that is the case, isn't it better to not display an empty asset/component?

Reply:
Done!